src/redux/header/tests/header.sagas.test.js

```diff
@@ -0,0 +1,135 @@
+import { expectSaga } from 'redux-saga-test-plan';
+import { call } from 'redux-saga/effects';
+import { combineReducers } from 'redux';
+
+import {
+  handleAddHeader,
+  handleHeaderDelete,
+  handleHeaderLoad,
+  handleHeadersLoad,
+  handleHeaderUpdate
+} from '../header.sagas';
+
+import Header, { initialState } from '../header.reducer';
+import {
+  createHeader,
+  getAllHeaders,
+  getHeaderById,
+  updateHeader
+} from '../header.operations';
+
+import { headers, id, mockHeader } from './header.variables';
+import { config } from '../../../configs';
+import { handleSuccessSnackbar } from '../../snackbar/snackbar.sagas';
+
+import {
+  deleteHeader,
```

Comment:
you shouldn't use this function in this test

Reply:
thanks